src/registrar/assets/sass/_theme/_admin.scss

```diff
@@ -416,8 +423,40 @@ address.dja-address-contact-list {
         font-size: 0.875rem;
         color: var(--body-quiet-color);
     }
+
+    address button.usa-button__clipboard-link {
+        font-size: 0.875rem !important;
+    }
+
+    td button.usa-button__clipboard-link {
+        font-size: 0.875rem !important;
+    }
 }
 
 .errors span.select2-selection {
     border: 1px solid var(--error-fg) !important;
+}
+
+.admin-icon-group {
+    position: relative;
```

Comment:
Could you add a comment up here saying this is all alignment-magic to make the copy button appear in the right place? (I'm assuming that's what this is, but it's not super clear just by reading the code)